models/articles-model.js

```diff
@@ -33,4 +32,15 @@ function selectArticles(articles, comments){
     })
 }
 
-module.exports = { selectTopics, selectArticleById, selectArticles }
\ No newline at end of file
+function selectCommentsByArticleId(article_id) {
+   return db
+   .query(`SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC`, [article_id])
+   .then(({rows:comments}) => {
+    if(!comments.length){
+        return Promise.reject({ status: 404, msg: 'article does not exist' })
+      }
+    return comments;
+   })
+}
```

Comment:
There is an issue with the error handling on line 40...
Not every article in the database have comments attached to them, for example `article_id: 2` has no comments, but it's still a valid article and therefore not an error